### PR TITLE
Quiz 07 on parameters

### PR DESCRIPTION
**Pull request by alayek**
Do I include a solution to these as well? Or tests?

---

As per #20

### CODE REVIEW

File: quizzes/07_parameters.test.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+console.log(
+  // randGen();
+  // weatherOnDemand(arr, 'Bali', 'London')``
```
Comment (kentcdodds) [inline quizzes/07_parameters.test.js]: Does this work? `arr` should probably be defined before, or turn it into a function declaration that we can call to get `arr`. Also, I think that this line has a typo (and the line above). I want to be able to uncomment the line really quickly :)

File: quizzes/07_parameters.test.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  // generates a random integer betweeb 0 and n
+  // assume n is always positive
+  // randGen(n) can be at least 0 and at max n - 1
+  // If n is undefined, output can be either 0 or 1
+  // TODO: refactor with default parameter to handle n = undefined
+
+  return Math.floor(n * Math.random());
+}
+
+function weatherOnDemand(args) {
```
Comment (kentcdodds) [inline quizzes/07_parameters.test.js:16]: I think that I'd rather refactor than write if reasonable. Could we have an ES5 implementation in here that we could refactor?

**Review by kentcdodds — CHANGES_REQUESTED** [review]
Just a few changes on this one. Thanks!

File: quizzes/07_parameters.test.js (hunk 2 above)
Comment (alayek) [reply quizzes/07_parameters.test.js:16]: Added an ES5 implementation. Please check.

**Review by kentcdodds — APPROVED** [review]
Great! Thanks!

**Pull request merged**